src/backend/web/routes/opml.js

```diff
@@ -0,0 +1,28 @@
+const express = require('express');
+const opml = require('opml-generator');
+
+const router = express.Router();
+
+const header = {
+  title: 'OPML Feeds',
+  dateCreated: new Date(2020, 2, 9),
+  ownerName: 'Jordan',
+};
+
+const outlines = [
+  {
+    text: 'txt',
+    title: 'My Open Source Experience',
+    type: 'rss',
+    xmlUrl: 'https://jrdnlxopensource.blogspot.com/feeds/posts/default/-/open-source',
+    htmlUrl: 'https://jrdnlxopensource.blogspot.com/',
```

Comment:
According to the [documentation](https://github.com/azu/opml-generator#usage), it seems that `xmlUrl` and `htmlUrl` should be used with quotes:
```js
'xmlUrl': 'https://jrdnlxopensource.blogspot.com/feeds/posts/default/-/open-source',
'htmlUrl': 'https://jrdnlxopensource.blogspot.com/',
```